src/mpid/ch4/netmod/ofi/util.c

```diff
@@ -260,6 +260,32 @@ int MPIDI_OFI_control_handler(int handler_id, void *am_hdr,
 #define isDOUBLE_COMPLEX(x) ((x) == MPI_DOUBLE_COMPLEX || (x) == MPI_COMPLEX8 || \
                               (x) == MPI_C_DOUBLE_COMPLEX)
 
+#undef FUNCNAME
+#define FUNCNAME check_mpi_acc_valid
+#undef FCNAME
+#define FCNAME MPL_QUOTE(check_mpi_acc_valid)
+MPL_STATIC_INLINE_PREFIX bool check_mpi_acc_valid(MPI_Datatype dtype, MPI_Op op)
+{
+    bool valid_flag = FALSE;
```

Comment:
`false`

Reply:
I was a little confused as most places use `TRUE` and `FALSE` in current master. But it looks like `bool` is defined in `mpl_base.h` and `true` and `false` are defined there. Will change to lowercase.

Reply:
done